src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs

```diff
@@ -31,6 +31,125 @@ public new abstract class Arm64 : AdvSimd.Arm64
             public static new bool IsSupported { [Intrinsic] get { return false; } }
         }
 
+
+        ///  Abs : Absolute value
+
+        /// <summary>
+        /// svint8_t svabs[_s8]_m(svint8_t inactive, svbool_t pg, svint8_t op)
```

Comment:
nit: We've historically also included the primary underlying instruction that gets emitted for a given API.

For example, with `AdvSimd`:
```csharp
/// <summary>
/// float64x2_t vabsq_f64 (float64x2_t a)
///   A64: FABS Vd.2D, Vn.2D
/// </summary>
```

In this case, I'd expect something like `A64: ABS Zd.B, Pg/M, Zn.B` (assuming I got that mostly right)